### PR TITLE
csshX: add page

### PR DESCRIPTION
**Pull request by mdwoycheksps**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/osx/csshX.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# csshX
+
+> Cluster ssh tool for MacOS.
+> More information: <https://github.com/brockgr/csshx>.
+
+- Connect to multiple hosts:
+
+`csshX hostname1 hostname2`
```
Comment (schneiderl) [inline pages/osx/csshX.md]:
```suggestion
`csshX {{hostname1}} {{hostname2}}`
```

File: pages/osx/csshX.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Connect to multiple hosts with ssh key:
+
+`csshX user@hostname1 user@hostname2 '--ssh_args' '-i /path/to/ssh-key.pem'`
```
Comment (schneiderl) [inline pages/osx/csshX.md]:
```suggestion
`csshX {{user@hostname1}} {{user@hostname2}} '--ssh_args' '-i {{/path/to/ssh-key.pem}}'`
```

File: pages/osx/csshX.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Connect to pre-defined cluster from /etc/clusters:
+
+`csshX cluster1`
```
Comment (schneiderl) [inline pages/osx/csshX.md]:
```suggestion
`csshX {{cluster1}}`
```

File: pages/osx/csshX.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# csshX
+
+> Cluster ssh tool for MacOS.
```
Comment (schneiderl) [inline pages/osx/csshX.md]: should we have ssh in upper case?

File: pages/osx/csshX.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# csshX
+
+> Cluster SSH tool for MacOS.
+> More information: <https://github.com/brockgr/csshx>.
+
+- Connect to multiple hosts:
+
+`csshX {{hostname1}} {{hostname2}}`
+
+- Connect to multiple hosts with ssh key:
```
Comment (schneiderl) [inline pages/osx/csshX.md]:
```suggestion
- Connect to multiple hosts with SSH key:
```

File: pages/osx/csshX.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# csshX
+
+> Cluster SSH tool for MacOS.
+> More information: <https://github.com/brockgr/csshx>.
+
+- Connect to multiple hosts:
+
+`csshX {{hostname1}} {{hostname2}}`
+
+- Connect to multiple hosts with SSH key:
```
Comment (agnivade) [inline pages/osx/csshX.md]: with _a given_ SSH key

File: pages/osx/csshX.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+`csshX {{user@hostname1}} {{user@hostname2}} '--ssh_args' '-i {{/path/to/ssh-key.pem}}'`
```
Comment (agnivade) [inline pages/osx/csshX.md]: path/to/ssh_key.pem

File: pages/osx/csshX.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+
+- Connect to pre-defined cluster from /etc/clusters:
```
Comment (agnivade) [inline pages/osx/csshX.md]: to a pre-defined cluster from `/etc/clusters`

File: pages/osx/csshX.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+
+`csshX {{cluster1}}`
```
Comment (agnivade) [inline pages/osx/csshX.md]: cluster_name

**Comment by mdwoycheksps** [comment]
Thanks @agnivade and @schneiderl. If formatting looks correct to you, whoever has powers to merge can get this in.

**Pull request merged**